### PR TITLE
update kubeedge architecture diagram

### PR DESCRIPTION
**Pull request by vincentgoat**
Signed-off-by: vincentgoat <linguohui1@huawei.com>

**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Update kubeedge architecture diagram, adding MetaServer module and describe.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +57,7 @@ KubeEdge consists of cloud part and edge part.
 - [ServiceBus](https://kubeedge.io/en/docs/architecture/edge/servicebus): a HTTP client to interact with HTTP servers (REST), offering HTTP client capabilities to components of cloud to reach HTTP servers running at edge.
 - [DeviceTwin](https://kubeedge.io/en/docs/architecture/edge/devicetwin): responsible for storing device status and syncing device status to the cloud. It also provides query interfaces for applications.
 - [MetaManager](https://kubeedge.io/en/docs/architecture/edge/metamanager): the message processor between edged and edgehub. It is also responsible for storing/retrieving metadata to/from a lightweight database (SQLite).
+- MetaServer: MetaServer starts a HTTP server and acts as an edge api-server for Kubernetes operators. It proxies the Kubernetes resource request to the dynamic controller in the cloud, and if the node is offline, the MetaServer will request the edge local storage for a workaround.
```
Comment (wackxu) [inline README.md:60]: an HTTP server

File: README.md [hunk 2]
```diff
@@ -47,7 +47,8 @@ KubeEdge consists of cloud part and edge part.
 - [CloudHub](https://kubeedge.io/en/docs/architecture/cloud/cloudhub): a web socket server responsible for watching changes at the cloud side, caching and sending messages to EdgeHub.
 - [EdgeController](https://kubeedge.io/en/docs/architecture/cloud/edge_controller): an extended kubernetes controller which manages edge nodes and pods metadata so that the data can be targeted to a specific edge node.
 - [DeviceController](https://kubeedge.io/en/docs/architecture/cloud/device_controller): an extended kubernetes controller which manages devices so that the device metadata/status data can be synced between edge and cloud.
-
+- SyncController: periodically check data that persists on edge and cloud, trigger reconcile if necessary.
```
Comment (wackxu) [inline README.md:50]: meta data

Comment (wackxu) [reply README.md:50]: an extended kubernetes controller which is responsible for reliable data transmission between cloud and edge. It periodically check meta data that persists on edge and cloud, trigger reconcile if necessary.

**Comment by wackxu** [comment]
plugins in your neew diagram such as csidriver、 webhook、edgecontroller is a single process and is not part of cloudcore

**Comment by wackxu** [comment]
`edgeApplication` looks like a API not a component of kubeedge

**Comment by wackxu** [comment]
in edge node, the component that need list-watch usally is CNI、CSI or some agent. we call this k8s operator is inaccurate

**Review by fisherxu — COMMENTED** [review]
What's meaning of Resource Runtime in Plugins?

**Comment by vincentgoat** [comment]
> What's meaning of Resource Runtime in Plugins?

For the EdgeApplication and NodeGroup controller runtime, called Resource Runtime here.
Maybe there is another appropriate name. What's your opinion?

**Pull request closed without merging**